controllers/managedocs_controller_test.go

```diff
@@ -1959,34 +1912,8 @@ var _ = Describe("ManagedOCS controller", func() {
 					}, timeout, interval).Should(BeTrue())
 				})
 			})
-			AfterAll(func() {
-				// Remove all resources for next deployment type
-				resList := []client.Object{
-					promTemplate.DeepCopy(),
-					promStsTemplate.DeepCopy(),
-					amTemplate.DeepCopy(),
-					amStsTemplate.DeepCopy(),
-					pdSecretTemplate.DeepCopy(),
-					grafanaDatasourceSecretTemplate.DeepCopy(),
-					amConfigTemplate.DeepCopy(),
-					podMonitorTemplate.DeepCopy(),
-					serviceMonitorTemplate.DeepCopy(),
-					promRuleTemplate.DeepCopy(),
-					ocsInitializationTemplate.DeepCopy(),
-					addonConfigMapTemplate.DeepCopy(),
-					addonParamsSecretTemplate.DeepCopy(),
-				}
-
-				for _, object := range resList {
-					k8sClient.Delete(ctx, object)
-				}
-
-				utils.EnsureNoResources(k8sClient, ctx, resList, timeout, interval)
-			})
 		})
 	}
 
-	runTests(convergedDeploymentType)
-	runTests(consumerDeploymentType)
-	runTests(providerDeploymentType)
+	runTest()
```

Comment:
Why do we need a function call? Can we just unwrap and put the context here? (the way it was before we added the `runTests` functions?

Reply:
- only a single reason, huge diff due to de-tab after removing function and important changes will not be evident
- I'm thinking to raise another pr just removing this anon function, I can add another commit if that suffices

Reply:
Why do you care about the diff size? I would argue that we should care about having each PR focused on a single coherent change - in this case, you are already touching the function and removing all the calls to it, so making the last step of inlining it just make sense